platform/osgi/pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>platform</artifactId>
+        <groupId>ddf.platform</groupId>
+        <version>2.11.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>pom</packaging>
+
+    <name>DDF :: Platform :: OSGi</name>
+    <artifactId>osgi</artifactId>
+
+    <!-- These versions should NOT change with the rest of DDF -->
+    <!-- but only if we upgrade Felix config admin (not recommended) -->
+    <properties>
```

Comment:
❓ As we upgrade Karaf, we need to remember to upgrade these versions. Is this going to become a maintenance problem?  We have this property in multiple places https://github.com/codice/ddf/blob/aaf326cbdd75823aeb85a490d48355230ecf8171/pom.xml#L199.  Looks like we might have forgotten to update it on one of the last karaf upgrades.  It looks like it should be 1.8.14 now.

Reply:
Fixed. No more props in the platform/osgi pom.